src/adapters/mc/OsConfigResource.c

```diff
@@ -168,10 +168,10 @@ void MI_CALL OsConfigResource_Load(
         CallMpiClose(g_mpiHandle, GetLog());
         g_mpiHandle = NULL;
     }
-    
-    AsbInitialize(GetLog());
 
     SetConsoleLoggingEnabled(false);
+    
+    AsbInitialize(GetLog());
```

Comment:
Does this change also need to be in `dev`?

Reply:
Will review there and fix if necessary.